src/app/admin/product-form/product-form.component.ts

```diff
@@ -0,0 +1,16 @@
+import { ProductService } from './../../product.service';
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-product-form',
+  templateUrl: './product-form.component.html'
+})
+export class ProductFormComponent {
+  categories: any[] = ['Bread', 'Dairy', 'Fruits', 'Seasonings', 'Vegetables'];
```

Comment:
this is an array of strings so categories : String[]. Avoid using any when possible